src/usage_metrics/models.py

```diff
@@ -65,6 +65,98 @@
     Column("remote_ip_full_location", String),
 )
 
+core_s3_logs = Table(
+    "core_s3_logs",
+    usage_metrics_metadata,
+    Column("request_id", String, primary_key=True, comment="A unique ID for each log."),
+    # Query information
+    Column("time", DateTime),
+    Column("request_uri", String),
+    Column("operation", String),
+    Column("bucket", String),
+    Column("bucket_owner", String),
+    Column("requester", String),
+    Column("http_status", Integer),
+    Column("bytes_sent", Integer),
+    # IP location
+    Column("remote_ip", String),
+    Column("remote_ip_city", String),
+    Column("remote_ip_loc", String),
+    Column("remote_ip_org", String),
+    Column("remote_ip_hostname", String),
+    Column("remote_ip_country_name", String),
+    Column("remote_ip_asn", String),
+    Column("remote_ip_country", String),
+    Column("remote_ip_timezone", String),
+    Column("remote_ip_latitude", Float),
+    Column("remote_ip_longitude", Float),
+    Column("remote_ip_postal", String),
+    Column("remote_ip_region", String),
+    Column("remote_ip_full_location", String),
+    # Other reported context
+    Column("access_point_arn", String),
+    Column("acl_required", String),
+    Column("authentication_type", String),
+    Column("cipher_suite", String),
+    Column("error_code", String),
+    Column("host_header", String),
+    Column("host_id", String),
+    Column("key", String),
+    Column("object_size", Float),
+    Column("referer", String),
+    Column("signature_version", String),
+    Column("tls_version", String),
+    Column("total_time", Integer),
+    Column("turn_around_time", Float),
+    Column("user_agent", String),
+    Column("version_id", String),
+)
+
+out_s3_logs = Table(
+    "out_s3_logs",
+    usage_metrics_metadata,
+    Column("request_id", String, primary_key=True, comment="A unique ID for each log."),
+    # Query information
+    Column("time", DateTime),
+    Column("table", String),
+    Column("version", String),
+    # IP location
+    Column("remote_ip", String),
+    Column("remote_ip_city", String),
+    Column("remote_ip_loc", String),
+    Column("remote_ip_org", String),
+    Column("remote_ip_hostname", String),
+    Column("remote_ip_country_name", String),
+    Column("remote_ip_asn", String),
+    Column("remote_ip_country", String),
+    Column("remote_ip_timezone", String),
+    Column("remote_ip_latitude", Float),
+    Column("remote_ip_longitude", Float),
+    Column("remote_ip_postal", String),
+    Column("remote_ip_region", String),
+    Column("remote_ip_full_location", String),
+    # Other reported context
+    Column("access_point_arn", String),
+    Column("acl_required", String),
+    Column("authentication_type", String),
+    Column("bytes_sent", Integer),
```

Comment:
Hm! While we won't be able to see the *exact* byte range, we'll be able to see when a range header successfully limited the amount of bytes transferred for our end users. Cool!